create/approve.py

```diff
@@ -17,14 +17,15 @@
 from ocflib.misc.shell import prompt_for_new_password
 from ocflib.misc.shell import red
 from ocflib.misc.shell import yellow
+from ocflib.ucb.groups import group_by_oid
 
 TEMPLATE = dedent(
     """\
-    user_name:
-    group_name:
-    callink_oid:
-    signatory:
-    email:
+    user_name: {empty}
+    group_name: {group_name}
+    callink_oid: {callink_oid}
+    signatory: {empty}
+    email: {email}
```

Comment:
I should have left a line item as well, but it's to put a space after the colon. Trailing spaces are caught by the linter.